app/serializers/search/reaction_serializer.rb

```diff
@@ -0,0 +1,32 @@
+module Search
+  class ReactionSerializer
+    include FastJsonapi::ObjectSerializer
+
+    attributes :id, :category, :status, :user_id
+
+    attribute :reactable do |reaction|
+      reactable = reaction.reactable
+      tags = if reactable.is_a?(Article)
```

Comment:
what about `reactable.respond_to?(:tags)` instead of checking the type? If we ever add `.tags` to comments, this will automagically work

Reply:
OH snap! That's brilliant!